#The message that will be sent to all_smartwifi_com. This will inform all of the vehicle's
#of every other vehicle with a smartwifi module.

int32 vehicle_id

float64 x
float64 y

float64 yaw

int32 back_connection_id